dataactvalidator/config/sqlrules/c5_award_financial_2.sql

```diff
@@ -17,6 +17,12 @@ SELECT
     uri AS "uniqueid_URI"
 FROM award_financial
 WHERE submission_id = {0}
+    AND gross_outlay_amount_by_awa_cpe IS NOT NULL
+    AND (COALESCE(gross_outlays_undelivered_cpe, 0) > 0
+         OR COALESCE(gross_outlays_undelivered_fyb, 0) > 0
+         OR COALESCE(gross_outlays_delivered_or_cpe, 0) > 0
+         OR COALESCE(gross_outlays_delivered_or_fyb, 0) > 0
+    )
```

Comment:
Would these always be checking for greater than zero? Do we just want to check for <> 0 to account for possible negatives? (not sure if that's something that should ever happen, probably not, but the situation should probably be accounted for)

Reply:
Good question, I don't think they should have negative values but we certainly should account for them.